scripts/release_tools/import_prod.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+
+# Imports a given staging area to the production DCP2 dataset
+# Prerequisites:
+#   * a working installation of the argo CLI tool (https://argoproj.github.io/argo-workflows/cli/)
+#   * access to the broad-dsp-monster-hca-prod google project
```

Comment:
disambiguation nitpicking:
* access to the project how? with the currently logged in gcloud cli account? do the credentials have to be application default?
* you'll also need `gcloud` and  `kubectl` installed
* `gdate` comes from GNU coreutils, so make sure to mention that people need that installed (`brew install coreutils`)
* this script relies on relative paths, so it will fail if run in a working directory other than the directory it's located in. we should either note that in the prerequisites or alter it to handle its working directory automatically.